scripts/jenkins/groovy/defineTestStages.groovy

```diff
@@ -357,6 +357,12 @@ def call(final pipelineContext) {
     [ // These run with reduced number of file descriptors for early detection of FD leaks
       stageName: 'XGBoost Stress tests', target: 'test-pyunit-xgboost-stress', pythonVersion: '3.5', timeoutValue: 40,
       component: pipelineContext.getBuildConfig().COMPONENT_PY, customDockerArgs: [ '--ulimit nofile=100:100' ]
+    ],
+    [
+      stageName: 'Kubernetes', target: 'test-h2o-k8s', timeoutValue: 20,
+      component: pipelineContext.getBuildConfig().COMPONENT_JAVA,
+      image: "${pipelineContext.getBuildConfig().DOCKER_REGISTRY}/opsh2oai/h2o-3-k8s:${pipelineContext.getBuildConfig().K8S_TEST_IMAGE_VERSION_TAG}",
+      customDockerArgs: ['-v /var/run/docker.sock:/var/run/docker.sock', '--network host'], dind: true
```

Comment:
took me a while to fingure out that dind is "docker in docker", since we use pretty descriptive param names I would also suggest naming this more verbosely like enableDockerInDocker

Reply:
This is a test currently, please wait for me to try it out.

Reply:
Sorry for the inconvenience.

Reply:
nit: I would name it `addToDockerGroup`, or something like that. In fact it's just adding the user in container to `docker` group, it has nothing to do with dind.

Reply:
Renamed.